components/commons/package.json

```diff
@@ -13,7 +13,7 @@
   "bugs": {
     "url": "https://github.com/adobe/spectrum-css/issues"
   },
-  "main": "dist/index-vars.css",
+  "main": "index.css",
```

Comment:
Can you tell me about this change?

Reply:
Yes, so commons doesn't actually have a compiled output because it's a pre-compilation utility with extends in it. When someone tries to access `@spectrum-css/commons` it was sending them to a non-existent package and so would always throw an error. Now if you want to use our extends in another project, you can `require("@spectrum-css/commons")` and load up all the extends we define here.

Reply:
Cool cool, thanks for explaining 🙂